airflow/models/dagrun.py

```diff
@@ -31,21 +31,26 @@
 from airflow.ti_deps.dep_context import DepContext
 from airflow.ti_deps.dependencies import SCHEDULEABLE_STATES
 from airflow.utils import timezone
-from airflow.utils.jobs_helpers import BACKFILL_JOB_PREFIX
 from airflow.utils.log.logging_mixin import LoggingMixin
 from airflow.utils.session import provide_session
 from airflow.utils.sqlalchemy import UtcDateTime
 from airflow.utils.state import State
 
 
+class DagRunType:
+    """Class with DagRun types"""
+    BACKFILL_JOB = "backfill_"
+    SCHEDULED = "scheduled__"
```

Comment:
@mik-laj it seems this won't work. Travis is happy but `airflow db reset` no.

Reply:
So maybe it's the right time to introduce the fix we talked about with Ash. There should be a method "load_all_models()" that should import all models from the DB (note ! just import - do nothing else). This method should be called in resetdb() so that all models can be loaded into sqlalchemy and register there. Note that there is a hack already implemented by me where I artificially add two deleted tables (USERS and another one) to the sqlalchemy so that all migration scenarios should work - these two lines should also be in the "load_all_models()" method.

Reply:
I would prefer to do this in separate PR. When I remove the last fixup everything works and all we need is place for one constant, that should be fixed as JIRA describes...

Reply:
If you move the "DagRunType" class to a separate file, the problem should disappear.  Now even more cyclical imports have been created, because many classes depend on DagRun.

Reply:
Done. Moved this to `airflow/utils/types.py`.